test/NuGet.Core.FuncTests/NuGet.Packaging.FuncTest/SigningTests/PackageSignatureVerifierTests.cs

```diff
@@ -194,7 +194,8 @@ public async Task VerifySignaturesAsync_ExpiredCertificateAndTimestamp_SuccessAs
                 }
             }
 
-            [CIOnlyFact]
+            https://github.com/NuGet/Home/issues/11459
```

Comment:
This probably fails the build?

Reply:
Sorry, haven't saved the change before pushing. Just fixed this.